src/style/palette/material.rs

```diff
@@ -523,84 +523,300 @@ pub const GRAY: NonAccentedPalette = NonAccentedPalette::from_variants(variants:
 pub const BLUE_GRAY: NonAccentedPalette = NonAccentedPalette::from_variants(variants::BLUE_GRAY);
 
 // Black and white included for completeness
-pub const BLACK: Color = Color::from_u32(0x000000);
-pub const WHITE: Color = Color::from_u32(0xFFFFFF);
+pub const BLACK: Color = Color::from_u32(0x0000_0000);
+pub const WHITE: Color = Color::from_u32(0x00FF_FFFF);
```

Comment:
And here - same question - I think we should allow this format for the entire file as the 3 byte format is better than the 4 byte formats (search-ability / readability / vertical space / etc.).